src/natcap/invest/ndr/ndr.py

```diff
@@ -1508,16 +1496,6 @@ def _calculate_load(
     eff_key = f'eff_{nutrient_type}'
     load_type_key = f'load_type_{nutrient_type}'
 
-    # Raise ValueError if unknown load_type
```

Comment:
I'm surprised I didn't have this in the validate function instead. I can't think of a reason why I would have preferred it here except to also benefit Python users. Huh.